apps/web/src/app/api/youtube/create-stream/route.ts

```diff
@@ -0,0 +1,163 @@
+import { NextRequest, NextResponse } from "next/server";
+import { getServerSession } from "next-auth/next";
+import { google } from "googleapis";
+import type { NextAuthOptions } from "next-auth";
+import GoogleProvider from "next-auth/providers/google";
+
+// Define auth options inline to match the configuration in [...nextauth]/route.ts
+const authOptions: NextAuthOptions = {
```

Comment:
use authOptions already defined in [...nextAuth]/route.ts file. export from there and use.